### DISCUSSION TITLE
Bug report: Edge outside transformationpane

### DISCUSSION THREAD

**Discussion by nolderosw** [bcakmakoglu/vue-flow#284] [Q&A] [answered]
Anyone got this?

![image](https://user-images.githubusercontent.com/21163236/190242914-891b3d5c-eced-4663-b6ea-bfa27f0b1883.png)

![image](https://user-images.githubusercontent.com/21163236/190242978-706fb14b-de54-411b-bf67-9d66247f7ac3.png)

**Comment by bcakmakoglu** [comment, marked as answer]
Can you check and see if this CSS-Rule is applied?

```css
.vue-flow__edges {
  overflow: visible;
}
```

It might be that either your browser or some custom styles / css-framework overwrote it.

**Reply by nolderosw** [reply]
Omg, thank you!!!!! I was 2 hours trying to solve! Can u tell me why that happens?

**Reply by bcakmakoglu** [reply]
I think that some browsers just use `overflow: hidden` as the default for svgs.
Why that is, I have no idea 😄 

Probably because support was lacking with IE but nowadays it works on modern browsers.

**Reply by nolderosw** [reply]
U rock, thanks again!
